### PR TITLE
Draft updates 24 april

### PR DESCRIPTION
**Pull request by ritikarawlani**
Closes # 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed
- [ ] I have selected a committee co-chair to review the PR

### CODE REVIEW

File: ch-37.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
           </div>
           <div class="top-bar-right hide-for-small-only">
             <ul class="menu">
-              <li class="menu-text tf-version">IHE IT Infrastructure (ITI) Technical Framework, Volume <span id="volumeNo"></span><br /><mark>Revision 20.0, August 4, 2023 – Final Text</mark></li>
+              <li class="menu-text tf-version">IHE IT Infrastructure (ITI) Technical Framework, Volume <span id="volumeNo"></span><br />Revision 21.0, April 24, 2024 – Final Text</li>
```
Comment (ritikarawlani) [inline ch-37.html:31]: revert it

File: ch-5.10.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
           </div>
           <div class="top-bar-right hide-for-small-only">
             <ul class="menu">
-              <li class="menu-text tf-version">IHE IT Infrastructure (ITI) Technical Framework, Volume <span id="volumeNo"></span><br /><mark>Revision 1.0, March 4, 2024 – Initial Draft</mark></li>
+              <li class="menu-text tf-version">IHE IT Infrastructure (ITI) Technical Framework, Volume <span id="volumeNo"></span><br />Revision 1.0, April 24, 2024 – Initial Draft</li>
```
Comment (ritikarawlani) [inline ch-5.10.html:29]: revert the change

File: ch-5.10.html [hunk 3]
```diff
@@ -156,8 +196,8 @@ <h5 id="5.10.3.1.1">5.10.3.1.1 "sigD" Header Parameter</h5>
           <li>sigD parameter shall be included as per 5.2.8.1 of the JAdES Specification</li>
             <li>mID member shall be present and set to "http://uri.etsi.org/19182/ObjectIdByURIHash"</li>
             
-              <li class="bullet-list1"><mark>
-                pars member shall shall be used and hold the document uniqueID. For documents that do not use a URI as the uniqueId, the Affinity Domain should determine an appropriate way to encode the DocumentEntry.uniqueId</mark>
+              <li class="bullet-list1">
+                The pars member shall be an array of strings that contain references to each data object being signed. This array is considered the manifest of the data objects being signed. Each string in this array shall be an URI. See <a href="5.10.6.1.9">Section 5.10.6.1.9</a> for more details.
```
Comment (ritikarawlani) [inline ch-5.10.html:200]: inline note: 

then we can say that the binary representation such documents are referred to as Data Objects which are the objects on which the digital signatures are calculated and verified

File: README.md [hunk 4]
```diff
@@ -24,17 +24,19 @@ It is presumed that the deviations from profile will yield implementations non c
...
+   "Note that Content Creators and Content Consumers should be capable of being configured to other conformance policies to support local policy. For example, some environments may choose a different JAdES profile, hashing algorithm, policy identifier, or signature purpose vocabulary. Content Creators would thus create Digital Signature blocks that are not conformant to this profile. Content Consumers can validate these Digital Signature blocks, and be capable of configured behavior according to the local policy. Deviations from these guidelines would need to be expressed in site policy and would be enumerated in the JWS-Signature block. For example, some environments may choose a different hashing algorithm, policy identifier, or signature purpose vocabulary. Some regions also require conformance to ISO 17090, which includes additional Certificate issuing, content, and validation rules."
```
[the remark this answers is not in the document]
Comment (ritikarawlani) [reply README.md:27]: This is in the ReadMe, stating an Open Issue that we removed any content and paragraphs about deviations...so the para above is removed content

File: ch-5.10.html (hunk 2 above)
Comment (ritikarawlani) [reply ch-5.10.html:29]: keeping it since ch 5.10 is a new addition

File: ch-5.10.html [hunk 5]
```diff
@@ -230,12 +270,15 @@ <h5 id="5.10.6.1.7">5.10.6.1.7 XDSDocumentEntry.title</h5>
...
       <h4 id="5.10.6.3">5.10.6.3 Document Sharing - Folder Metadata</h4>
       <p>This document content profile makes no changes to the structure of Folders.</p>
       <h4 id="5.10.6.4">5.10.6.4 Document Associations</h4>
-      <p>When Detached Signature Option is used, the Content Creator shall use the “SIGNS” associationType Document Relationship to associate the signature (sourceObject) to the documents that it signs (targetObjects). <mark>See Section 4.2.2.</mark></p>
+      <p>When Detached Signature Option is used, the Content Creator shall use the “SIGNS” associationType Document Relationship to associate the signature (sourceObject) to the documents that it signs (targetObjects). See ebRIM Representation <a href="https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.2">Section 4.2.2</a>.</p>
       <h3 id="5.10.7">5.10.7 Security Considerations</h3>
-      <p><mark>See JAdES specification for risk assessment and mitigation plan on Digital Signatures.</mark></p>
+      <p>See ETSI ES 201 733 and ETSI TS 119 511 specifications for risk assessment and mitigation plan on Digital Signatures.</p>
```
[the remark this answers is not in the document]
Comment (ritikarawlani) [reply ch-5.10.html:281]: I see this is an assessment framework for ETSI TS 119 511 -> are you suggesting I include this as an informative reference?

File: ch-5.10.html [hunk 6]
```diff
@@ -156,12 +201,13 @@ <h5 id="5.10.3.1.1">5.10.3.1.1 "sigD" Header Parameter</h5>
           <li>sigD parameter shall be included as per 5.2.8.1 of the JAdES Specification</li>
             <li>mID member shall be present and set to "http://uri.etsi.org/19182/ObjectIdByURIHash"</li>
             
-              <li class="bullet-list1"><mark>
-                pars member shall shall be used and hold the document uniqueID. For documents that do not use a URI as the uniqueId, the Affinity Domain should determine an appropriate way to encode the DocumentEntry.uniqueId</mark>
+              <li class="bullet-list1">
+                The pars member shall be an array of strings that contain references to each data object* being signed. This array is considered the manifest of the data objects being signed. Each string in this array shall be an URI. See <a href="5.10.6.1.9">Section 5.10.6.1.9</a> for more details.
              </li>
             <li>ctys member shall be present</li>
             <li>hashV, and the hashM members shall be present</li>
```
Comment (JohnMoehrke) [inline ch-5.10.html:208]: I thought it was either hashV xor hashM. How and why would we require both?

Comment (JohnMoehrke) [reply ch-5.10.html:208]: actually I am misunderstanding the hashV... Why are we setting b64 to true? This means that the document bits need to be base64url encoded before it is hash, right? Why do we want that? Why not just hash the document?

Comment (ritikarawlani) [reply ch-5.10.html:208]: The hashM member shall be a string identifying a digest algorithm. 

The hashV member shall be a non-empty array of strings. Each element of the array shall contain: 1) The base64url-encoded digest value of the data object referenced by the parameter value that is present in the same position of the pars array if the b64 header parameter is present and set to "false". 

2) The base64url-encoded digest value of the base64url-encoded data object referenced by the parameter value that is present in the same position of the pars array if the b64 header parameter is absent or it is present and set to "true". 

as per 5.2.8.3.3 Mechanism ObjectIdByURIHash  of JAdES both HashM and HashV shall be present. 

re: b64 please read rows 9 and 10 in this excel https://docs.google.com/spreadsheets/d/1nfjZqhJuA4PxbxZAOxa5DEgH4_SPrHZZ2zzACCUMe6Y/edit#gid=0 

b64 not only decides hashV but also what gets signed (generally) , but in our case for detached, the payload is not included in the signature, for enveloping it is, and shall be true

Comment (ritikarawlani) [reply ch-5.10.html:208]: @litlfred your thoughts?

Comment (JohnMoehrke) [reply ch-5.10.html:208]: thanks. That helped.

I am still unclear why we would pick to set b64 to true? I am not suggesting it be false, but I am unclear what the benefit of true is, or what the reasoning for picking true is.
